### ISSUE TITLE
Error parsing date column in JRuby

### ISSUE THREAD

**Issue by duncan-bayne** [pythonicrubyist/creek#16]
I have an XLSX spreadsheet that parses correctly using creek on MRI.  When I parse the same file using creek on JRuby, I get the following error:

```
[1] pry(main)> require 'creek'
Ignoring jruby-launcher-1.1.1-java because its extensions are not built.  Try: gem pristine jruby-launcher --version 1.1.1
Ignoring jruby-launcher-1.1.1-java because its extensions are not built.  Try: gem pristine jruby-launcher --version 1.1.1
=> true
[2] pry(main)> book = Creek::Book.new('./test.xlsx')
=> #<Creek::Book:0x34a75079 @files=./test.xlsx, @shared_strings=#<Creek::SharedStrings:0x107ed6fc @book=#<Creek::Book:0x34a75079 ...>>>
[3] pry(main)> book.sheets[0].rows.each { |row| }
RangeError: too big for int: 2165046705
from /home/duncan/.rbenv/versions/jruby-1.7.19/lib/ruby/1.9/date.rb:1354:in `+'
```

Versions:

* creek (1.0.9)
* nokogiri (~> 1.6.0)
* rubyzip (>= 1.0.0)
* jruby 1.7.19 (1.9.3p551) 2015-01-29 20786bd on Java HotSpot(TM) 64-Bit Server VM 1.8.0_45-b14 +jit [linux-amd64]

You can download the test XLSX file [here](http://live25percentlonger.com/test.xlsx).

**Comment by pythonicrubyist** [comment]
@duncan-bayne  Thanks for using this gem. I have never tested creek on JRbuy, but lets fix this issue.
I tried to open up your test Excel file in MS Excel for mac Version 2011 and I got the following error while opening the file:

![screen shot 2015-09-04 at 11 57 22 pm](https://cloud.githubusercontent.com/assets/1863594/9695373/46584da0-5361-11e5-8d51-24513c40fc07.png)

After repairing the file, the opened file is filled with garbage data.
Are you sure the Excel file is in a right format?
